src/conky.cc

```diff
@@ -584,10 +584,10 @@ void human_readable(long long num, char *buf, int size) {
   }
   if (short_units.get(*state)) {
     width = 5;
-    format = "%.*f%.1s";
+    format = "%.*f %.1s";
   } else {
     width = 7;
-    format = "%.*f%-.3s";
+    format = "%.*f %-.3s";
```

Comment:
If you're adding an extra character, would you not also need to increase the width?

Reply:
Not sure. It's locally working as far as I can see. I can certainly add the extra, if you'd like.

Reply:
Actually, never mind. I forgot what the `width` argument did with `spaced_print()`, and after review it I can see it applies to the width of the value being printed, so it's fine in this case.